New version: Bijectors v0.13.18

JuliaRegistrator opened the pull request:
- Registering package: Bijectors
- Repository: https://github.com/TuringLang/Bijectors.jl
- Created by: @mhauru
- Version: v0.13.18
- Commit: 70fb426a8b86c9321e647838cb209eac40387db9
- Reviewed by: @mhauru
- Reference: https://github.com/TuringLang/Bijectors.jl/commit/70fb426a8b86c9321e647838cb209eac40387db9#commitcomment-146100016
- Description: Implementation of normalising flows and constrained random variable transformations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd65e0f1feee54543f284b04a7d07100d7bb20e6af7124850cd8f0c7fdcfe53fd72c3d111138d42337960fe60cd8605aa07a236537981f056f162c780f993bcd6cffce2d98e1edb41f607e5abe2fda24df4215039baca6b9d62c62d31f0bfb5c10a96b6e86ebb6e251d1c903cf7a9b4736ed78cd05c8b74a052139299232952fc0a3d26d642aec20d74748a5acdec094dedf2cb11bb6887239009bb3ee419ac6addb48fc780b64e9a4f8bb85d5edb770c25 -->

mhauru commented:
Replaced by https://github.com/JuliaRegistries/General/pull/113131, please close.

JuliaRegistrator committed:
New version: Bijectors v0.13.18

UUID: 76274a88-744f-5084-9051-94815aaf08c4
Repo: https://github.com/TuringLang/Bijectors.jl.git
Tree: 92edc3544607c4fda1b30357910597e2a70dc5ea

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

mhauru commented:
[noblock] Let's see if leaving a new comment will unblock this after editing the first comment.
